tools/exporter/exporter.py

```diff
@@ -48,8 +48,17 @@
 # When exporting actions to CSV, if property is None, print this value to CSV
 EMPTY_RESPONSE = ''
 
+# accepted configuration options for hosting region.
```

Comment:
This is duplicated here. Can it be made part of the SDK in one place and imported from here?